asset/scss/style.scss

```
@@ -0,0 +1,381 @@
+@import "variable";
+
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+  font-family: "Montserrat", sans-serif;
+}
+
+ul {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+// mixin
+@mixin bgcolor {
+  background-color: $bg-color;
+  color: $white-color;
+}
+
+// main
+.main {
+  line-height: 1;
+  max-width: 1920px;
+}
+.title--left {
+  font-weight: 200;
+}
+.title--right {
+  font-weight: 800;
+}
+
+// header
+.header {
+  @include bgcolor();
+}
+.header-wrap {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 1170px;
+  margin: 0 auto;
+}
+
+.navbar {
+  display: flex;
+}
+.navbar-logo {
+  display: inline-block;
+  padding: 21px 0;
+}
+
+.navbar-items {
+  border: 2px solid transparent;
+
+  a {
+    display: inline-block;
+    color: $white-color;
+    font-size: 1.4rem;
+    font-weight: 700;
+    padding: 6px 16px;
+    transition: all 0.3s;
+
+    &:hover {
+      color: $primary-color;
+    }
+  }
+}
+.active-nav {
+  border-color: $white-color;
```

```suggestion
  border: 2px solid $white-color;
```
xử lý thêm khi di chuột vào sẽ đổi màu border thành $primary-color